lib/pure/collections/deques.nim

```diff
@@ -86,7 +86,7 @@ proc initDeque*[T](initialSize: int = defaultInitialSize): Deque[T] =
   ## * `toDeque proc <#toDeque,openArray[T]>`_
   result.initImpl(initialSize)
 
-proc toDeque*[T](x: openArray[T]): Deque[T] {.since: (1, 3).} =
+proc toDeque*[T](x: sink openArray[T]): Deque[T] {.since: (1, 3).} =
```

Comment:
```suggestion
proc toDeque*[T](x: openArray[T]): Deque[T] {.since: (1, 3).} =
```

Please don't use `sink openArray` until we sorted out this part of the spec.